core_backend/app/llm_call/parse_input.py

```diff
@@ -122,15 +126,15 @@ async def wrapper(
         """
         Wrapper function to translate the question.
         """
-        question, response = _translate_question(question, response)
+        question, response = await _translate_question(question, response)
         response = await func(question, response, *args, **kwargs)
 
         return response
 
     return wrapper
 
 
-def _translate_question(
+async def _translate_question(
```

Comment:
We're sometimes adding "async" in the name of the function and sometimes not - should we always add or always remove (since we're making every function async anyway)?

Reply:
We should only use it when it is awaiting something in the function. It would be misleading otherwise and I assume adds an ayncio overhead